Porting/perlgov-team-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env perl
+use v5.30.0;
+use warnings;
+use JSON::PP;
+
+my $file = 'Porting/core-team.json';
+my $data = JSON::PP->new->decode(scalar `cat $file`);
+
+my $pod = q{};
+
+for my $key (qw( active inactive )) {
+  $pod .= qq{=head2 \u$key Members\n\n=over 4\n\n};
+
+  my @items = map {; "<$_>" } $data->{$key}->@*;
+
+  open(my $fh, '-|', 'git', 'check-mailmap', @items)
```

Comment:
These args should be encoded back to UTF-8.

Reply:
This still needs to be done with the input now being decoded. Pipe open takes byte arguments.